app/javascript/packages/document-capture/components/acuant-selfie-camera.tsx

```diff
@@ -0,0 +1,144 @@
+import React, { useContext, useEffect } from 'react';
```

Comment:
I don't think you should need to import `React` directly here.

```suggestion
import { useContext, useEffect } from 'react';
```